src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10501,84 +10492,89 @@ static ColliderQuadInit D_808546A0 = {
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
 
-void func_8084663C(Actor* thisx, PlayState* play) {
+void Player_DoNothing(Actor* thisx, PlayState* play) {
 }
 
-void Player_StartMode0(PlayState* play, Player* this) {
-    this->actor.update = func_8084663C;
+void Player_StartMode_Nothing(PlayState* play, Player* this) {
+    this->actor.update = Player_DoNothing;
     this->actor.draw = NULL;
 }
 
-void Player_StartMode2(PlayState* play, Player* this) {
+void Player_StartMode_BlueWarp(PlayState* play, Player* this) {
     Player_SetupAction(play, this, Player_Action_8084F710, 0);
+
     if ((play->sceneId == SCENE_LAKE_HYLIA) && IS_CUTSCENE_LAYER) {
         this->av1.actionVar1 = 1;
     }
+
     this->stateFlags1 |= PLAYER_STATE1_29;
     LinkAnimation_Change(play, &this->skelAnime, &gPlayerAnim_link_okarina_warp_goal, 2.0f / 3.0f, 0.0f, 24.0f,
                          ANIMMODE_ONCE, 0.0f);
     this->actor.world.pos.y += 800.0f;
 }
 
-static u8 D_808546F0[] = { ITEM_SWORD_MASTER, ITEM_SWORD_KOKIRI };
-
-void func_80846720(PlayState* play, Player* this, s32 arg2) {
-    s32 item = D_808546F0[(void)0, gSaveContext.save.linkAge];
-    s32 itemAction = sItemActions[item];
+void Player_TakeOutSword(PlayState* play, Player* this, s32 playSfx) {
```

Comment:
Is this generalizable to when Link takes out any sword during gameplay, or is this specific to certain cutscenes/scene transitions?

Reply:
This is only taking out, NOT using. In gameplay you USE sword which will take it out and swing it all in one go (which doesnt use this function, the item system usually does all this stuff). 
So essentially, it is only relevant in cutscenes

Reply:
Notably this will not play any animation and expects something else (like a cutscene) to already be handling that. If you want him to visibly appear to take out the sword, you have to call this function on the frame he reaches back to his sheath.

So, this can be `Player_PutSwordItemInHand` or something, idk. doesnt sound as great

Reply:
I went with `Player_PutSwordInHand` and added a function comment to clarify